Description: Bump convert_case to 0.11, as packaged in Debian
Author: Abhinaba Rakshit <abhinaba.rakshit@oss.qualcomm.com>
Forwarded: not-needed
Last-Update: 2026-10-08
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
Index: mink-idl-compiler/idlc_codegen_java/Cargo.toml
===================================================================
--- mink-idl-compiler.orig/idlc_codegen_java/Cargo.toml
+++ mink-idl-compiler/idlc_codegen_java/Cargo.toml
@@ -6,6 +6,6 @@ edition = "2021"
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [dependencies]
-convert_case = "0.6.0"
+convert_case = "0.11.0"
 idlc_codegen = { path="../idlc_codegen" }
 idlc_mir = { path="../idlc_mir" }
Index: mink-idl-compiler/idlc_codegen_rust/Cargo.toml
===================================================================
--- mink-idl-compiler.orig/idlc_codegen_rust/Cargo.toml
+++ mink-idl-compiler/idlc_codegen_rust/Cargo.toml
@@ -6,7 +6,7 @@ edition = "2021"
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [dependencies]
-convert_case = "0.6.0"
+convert_case = "0.11.0"
 idlc_codegen = { path="../idlc_codegen" }
 idlc_errors = {path = "../idlc_errors" }
 idlc_mir = { path="../idlc_mir" }
